Doc/library/pathlib.rst

```diff
@@ -1052,6 +1052,20 @@ call fails (for example because the path doesn't exist).
       Added return value, return the new Path instance.
 
 
+.. method:: Path.absolute()
+
+   Make the path absolute, without normalization or resolving symlinks, and return a new path
+   object.  This is equivalent to prepending the current directory::
+
+      >>> p = Path('..')
+      >>> p
+      PosixPath('..')
+      >>> p.absolute()
+      PosixPath('/home/antoine/pathlib/..')
+      >>> Path.cwd() / p
+      PosixPath('/home/antoine/pathlib/..')
```

Comment:
What is the purpose of this example? If it's an exception or specific detail then it should be specified in the documentation directly (along with any other potential quirks).

Reply:
The code block is introduced with:

```rest
This is equivalent to prepending the current directory::
```

So the last example re-enforces that point. But perhaps it's OTT?

Reply:
Yeah, I don't think that level of example detail is necessary. I would just remove it. I would just have an example showing how it takes the cwd and another path and just joins them.

Reply:
I've adjusted the docs, but it's possible I misunderstood you. Let me know if I got it wrong :D